tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/CIEnvironmentVariableTests.cs

```diff
@@ -134,6 +138,42 @@ public void CheckEnvironmentVariables(JsonDataItem jsonData)
             }
         }
 
+        [SkippableFact]
+        public void GithubEventJsonTest()
+        {
+            var reloadEnvironmentData = typeof(CIEnvironmentValues).GetMethod("ReloadEnvironmentData", BindingFlags.Instance | BindingFlags.NonPublic);
+
+            // Check if the CI\Data folder exists.
+            var ciDataFolder = DataHelpers.GetCiDataDirectory();
+
+            // JSON file path
+            var jsonFile = Path.Combine(ciDataFolder, "githubevent", "github-event.json");
+
+            // Let's test the github-event.json load and check the values first.
+            var githubEnvVars = new GithubActionsEnvironmentValues<DictionaryValuesProvider>(
+                new DictionaryValuesProvider(
+                    new Dictionary<string, string>
+                    {
+                        [CIEnvironmentValues.Constants.GitHubEventPath] = jsonFile,
+                    }));
+
+            reloadEnvironmentData?.Invoke(githubEnvVars, null);
```

Comment:
tiny nit: probably better to use `!` instead of `?` here as `reloadEnvironmentData` should never be `null` unless the test is broken, right?

Reply:
in both cases the test will fail :)